Src/BootCamp.Chapter/BalanceStats.cs

```diff
@@ -0,0 +1,374 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Text;
+using System.Text.RegularExpressions;
+
+namespace BootCamp.Chapter
+{
+    public class BalanceStats
+    {
+        string[] allNames { get; set; }
+        float[][] allBalances { get; set; }
+
+        public void ParsePeopleAndBalance(string[] peopleAndBalances)
+        {
+            if (string.IsNullOrEmpty(peopleAndBalances[0]))
+            {
+                return;
+            }
+
+            CultureInfo.CurrentCulture = CultureInfo.InvariantCulture;
```

Comment:
Side effect: InvariantCulture will be set all across the program. Instead, have parser methods that parse with the needed culture applied.